web-app/js/portal/details/SubsetItemsWrapperPanel.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.details');
+
+Portal.details.SubsetItemsWrapperPanel = Ext.extend(Ext.Panel, {
+
+    constructor: function(cfg) {
+
+        var tabPanelForLayer = new Portal.details.SubsetItemsTabPanel({
+            map: cfg.map,
+            layer: cfg.layer,
+            listeners: {
+                beforeTabChange: this._doTracking
+            }
+        });
+
+        this.layer = cfg.layer;
```

Comment:
This line shouldn't be needed. All values from config should be automatically applied to 'this'.